cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -50,6 +50,7 @@
     if node.chef_environment == 'staging' && node.name == 'staging' # 'real' staging only
       cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'import_google_sheets')
+      cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'sync_dropbox')
```

Comment:
To achieve near-realtime previewing of markdown changes in staging, I think we should aim to bump up the frequency!